test/integration/api/OrderAPI.spec.ts

```diff
@@ -742,7 +742,7 @@ describe('OrderAPI', () => {
       );
 
       const signerWalletAddress = ethUtil.bufferToHex(ethUtil.pubToAddress(signerPublicKey));
-      expect(signerWalletAddress).to.equal(DEFAULT_ACCOUNT);
+      expect(signerWalletAddress).to.equal(DEFAULT_ACCOUNT.toLowerCase());
```

Comment:
why do you have to do this?

Reply:
Web3 1.0 returns mixed case addresses, but old tools like ethUtil returns addresses that are lowercase vs mixed case. 

In our default accounts, we previously had lower case addresses, so all of our tests needed to map the accounts to lower case things.